orchagent/qosorch.cpp

```diff
@@ -303,23 +272,9 @@ bool DscpToTcMapHandler::removeQosItem(sai_object_id_t sai_object)
 
     if (sai_object == gQosOrch->m_globalDscpToTcMap)
```

Comment:
The entire `if` statement can be removed because it is guaranteed that no reference to the map being removed once the flow reaches here.
`removeQosItem` is called from `QosMapHandler::processWorkItem` where the reference is checked before calling.
Originally, we did not have a CONFIG_DB item to track the reference of the global map so we had to check it here.
Now we do have it in CONFIG_DB and the reference is tracked so no need to check it anymore.

Reply:
Good suggestion. I remove the variable now. Thanks